docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+services:
+  arma-reforger:
+    image: ghcr.io/acemod/arma-reforger:edge
+    platform: linux/amd64
+    container_name: arma-reforger
+    ports:
+      - "2001:2001/udp"
+      - "17777:17777/udp"
+    volumes:
+      - ./reforger/configs:/reforger/Configs
+      - ./reforger/profile:/home/profile
+    environment:
+      - SERVER_REGION=EU-NL 
+      - SERVER_HOST_REGISTER_ADDRESS=public-ip
+      - GAME_NAME=My Docker Reforger Server
+      - GAME_PROPS_DISABLE_THIRD_PERSON=true
```

Comment:
We shouldn't impose such defaults in this project. If want those here at all (which I would say no), they should be left as default in the Dockerfile.

Reply:
Agree, `GAME_PROPS_DISABLE_THIRD_PERSON` should be left out here

Reply:
We could, but that means another set of defaults to maintain outside of the Dockerfile. Way too easy to forget to change in both places.

Reply:
You are actually wrong there.  You can keep all these important vars in a single .env file.   Use this .env during build, and also use it in combination with a docker-compose.yml

Reply:
I'd say that introducing a .env file increases the complexity. Let's keep the PR like this, if you want to introduce a .env file you're free to create your own PR.